examples/MvcWeb/Controllers/ProjectApproveController.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MvcWeb.TheraLang.Entities;
+using MvcWeb.TheraLang.UnitOfWork;
+
+
+namespace MvcWeb.Controllers
+{
+    [Route("api/projects/approve")]
+    [ApiController]
+    public class ProjectApproveController : ControllerBase
```

Comment:
this should be ProjectController (if it exists - please put your code there if no - rename)